docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
    :width: 600
 
-**Modeling approach** 
-
-- Modeling tools like DisMod, ST-GPR
+**DisMod** 
+
+Disease Model Meta Regression 2.1 (DisMod, sometimes DisMod MR) is a statistical modeling tool developed for the Global
+Burden of Disease study to estimate non-fatal disease burden. It is the most frequently used tool for non-fatal modeling
+in the GBD and models that are run outside of it are frequently called *custom models*. 
+
+DisMod has a user interface. The 
+`External Version
+<https://vizhub.healthdata.org/epi/>`_
+shows published and final models for GBD 2017.
+The 
+`Internal Version
+<https://internal.ihme.washington.edu/epi/>`_
+is where new and ongoing modeling occurs and is for GBD 2019 (but shows GBD 2017 best models). 
+
+There are three main components to DisMod. The first is that it
+is a meta-regression statistical model. This means that it uses point estimates with uncertainty around those estimates and covariates
+to predict disease burden for every location/year/age/sex. The second is that it is a compartmental model of disease. This
+means that DisMod is solving differential equations while fitting meta-regression estimates to enforce consistency between
+disparate measures of disease like incidence, prevalence, remission, and excess mortality. The third is that DisMod is age-integrating.
```

Comment:
I think that this section could be numbered or bullet-ed to help keep it straight for the reader. Maybe bolded components that follow with a non-bold description of the component?

Reply:
I agree bullets or a numbered list would be good. You could also make it less wordy and make the 3rd point parallel in structure to the first two, e.g.

There are three main components to DisMod:

1. A meta-regression statistical model. This means...
2. A compartmental model of disease. This means...
3. Age integration. This means...

You could also try a definition list (rather than a numbered list) to implement Ali's suggestion of a bolded component followed by a description.

Reply:
👍